lib/versions_repository.py

```diff
@@ -25,3 +29,89 @@ def setup_versions_repository(config):
         raise
 
     return versions_repo
+
+
+def create_html_table(packages):
+    """
+    Create a HTML table with the packages versions
+
+    Args:
+        packages ([Package]): packages
+    """
+
+    table = etree.Element('table')
+    thead = etree.SubElement(table, 'thead')
+    software_th = etree.SubElement(thead, 'th')
+    software_th.text = 'Software'
+
+    version_th = etree.SubElement(thead, 'th')
+    version_th.text = 'Version'
+
+    tbody = etree.SubElement(table, 'tbody')
+
+    for package in packages:
+        tr = etree.SubElement(tbody, 'tr')
+        package_name_td = etree.SubElement(tr, 'td')
+        package_name_td.text = package.name
+
+        package_version_td = etree.SubElement(tr, 'td')
+        package_version_td.text = package.version
+
+    return etree.tostring(table, pretty_print=True)
+
+
+def replace_html_table(file_path, html_table):
+    """
+    Replace all HTML tables in a file with a HTML table
+
+    Args:
+        file_path(str): file path
+        html_table (str): string representation of the HTML table which will
+            replace existing ones
+    """
+
+    HTML_TABLE_TEMPLATE_VARIABLE = "%%HTML_TABLE%%"
+
+    with file(file_path, "r") as f:
+        lines = f.readlines()
+
+    new_lines = []
+    in_table = False
+    for line in lines:
+        if not in_table and "<table>" in line:
+            in_table = True
+            new_lines.append(HTML_TABLE_TEMPLATE_VARIABLE)
```

Comment:
Do you really need this constant?
You could append the `html_table` directly here, and continue to ignore the table lines.